opslevel/resource_opslevel_team.go

```diff
@@ -92,6 +113,14 @@ func (teamResource *TeamResource) Schema(ctx context.Context, req resource.Schem
 	resp.Schema = schema.Schema{
 		MarkdownDescription: "Team Resource",
 		Attributes: map[string]schema.Attribute{
+			"all_members": schema.ListAttribute{
+				Description: "Unordered list of all team members. Both managed by terraform and not.",
```

Comment:
Why unordered?  Couldn't we sort first in `teamMembersModelToListValue` since the input is a slice?

Reply:
@rocktavious unordered should be fine because we can rely on the API to order the members.

Reply:
my understanding is our API only orders list by "creation date" of the underlying row in the DB.  So if i remove a member "kyle" and then re add "kyle" back - that user will be the last entry in the list.  That HUGELY unexpected.  Lists should be sorted alphabetically always - either by Name, Alias or E-mail.